### PR TITLE
translate en to fr

### PR DESCRIPTION
**Pull request by Flayor**
It's done

### CODE REVIEW

File: Juice.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -598,7 +630,7 @@
 				CODE_SIGN_IDENTITY = "Mac Developer";
 				COMBINE_HIDPI_IMAGES = YES;
 				CURRENT_PROJECT_VERSION = 529;
-				DEVELOPMENT_TEAM = YN24FFRTC8;
+				DEVELOPMENT_TEAM = A925WRGBAZ;
```
Comment (brianmichel) [inline Juice.xcodeproj/project.pbxproj]: I can't accept these changes to the development team, sorry it need so to continue to be the old development team.

File: Juice.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -490,17 +500,38 @@
...
+			sourceTree = "<group>";
+		};
+		AA66D5581E0C77EE00D83161 /* GeneralPreferencesViewController.xib */ = {
+			isa = PBXVariantGroup;
+			children = (
+				AA66D5571E0C77EE00D83161 /* en */,
+				AA66D55B1E0C77F700D83161 /* fr */,
+			);
+			name = GeneralPreferencesViewController.xib;
+			path = ../../.Trash/Juice/Classes;
```
Comment (brianmichel) [inline Juice.xcodeproj/project.pbxproj]: This path seems wrong, should this be pointing to the file located within the project?

**Comment by brianmichel** [comment]
With this patch applied the project does not compile
screen shot 2016-12-23 at 9 09 38 am

**Comment by Flayor** [comment]
Ok it's done :)

**Comment by Flayor** [comment]
Merry Chrismas

I have a problem with encodage to pbproject i resolved that tomorow :)

**Comment by brianmichel** [comment]
@Flayor this still has the development team changes in the project file.

**Comment by brianmichel** [comment]
@Flayor hey sorry for not looking at this sooner I've been busy with work, but I'll take a look at this tomorrow evening but it's looking good!

**Comment by Flayor** [comment]
No problem take your time :)

**Comment by brianmichel** [comment]
Overall I think this looks okay, I'll merge it in and fix up anything that is broken within the `master` branch. Thanks for the translations!

**Comment by Flayor** [comment]
Your welcome ... i continue translate with your update ... just,

for the word "scale" want to tell you "theme" scale in french with the sentence doesn't mean anything

**Pull request merged**